packages/frontend/src/components/ProjectConnection/WarehouseForms/PostgresForm.tsx

```diff
@@ -68,228 +72,279 @@ const PostgresForm: FC<{
+                                                                    ? 'Copied'
+                                                                    : 'Copy'
+                                                            }
+                                                            withArrow
+                                                            position="right"
+                                                        >
+                                                            <ActionIcon
+                                                                color={
+                                                                    copied
+                                                                        ? 'teal'
+                                                                        : 'gray'
+                                                                }
+                                                                onClick={copy}
+                                                            >
+                                                                <MantineIcon
+                                                                    icon={
+                                                                        copied
+                                                                            ? IconCheck
+                                                                            : IconCopy
+                                                                    }
+                                                                />
+                                                            </ActionIcon>
+                                                        </Tooltip>
+                                                    )}
+                                                </CopyButton>
+                                            </>
+                                        }
+                                    />
+                                )}
+                                <Button
+                                    onClick={() => mutate()}
+                                    loading={isLoading}
+                                    disabled={disabled || isLoading}
+                                >
+                                    {sshTunnelPublicKey
+                                        ? 'Regenerate key'
+                                        : 'Generate public key'}
+                                </Button>
+                            </Stack>
+                        </FormSection>
+                    </Stack>
                 </FormSection>
-            </FormSection>
-            <AdvancedButtonWrapper>
-                <AdvancedButton
-                    icon={isOpen ? 'chevron-up' : 'chevron-down'}
-                    text={`Advanced configuration options`}
-                    onClick={toggleOpen}
-                />
-            </AdvancedButtonWrapper>
+                <Accordion
```

Comment:
I think we should keep using a simple button as we don't take advantage of the accordion functionality. A button with a variant "subtle" and a left icon would simplify the code.